Iterable-iOS-SDK/IterableAction.h

```diff
@@ -0,0 +1,29 @@
+//
+//  IterableAction.h
+//  Iterable-iOS-SDK
+//
+//  Created by Victor Babenko on 5/1/18.
+//  Copyright © 2018 Iterable. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+extern NSString *const IterableActionTypeDismiss;
+extern NSString *const IterableActionTypeOpen;
+extern NSString *const IterableActionTypeDeeplink;
+extern NSString *const IterableActionTypeTextInput;
```

Comment:
Are we not supporting silentActions as a type? Or is it going to be combined with Dismiss?

Reply:
It is combined with 'dismiss'. If it has something in 'data', that is executed as a silent action.
There was a request to have openUrl that opens the browser, we may want to add that too.